packages/infolists/src/Components/Concerns/HasHeaderActions.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
Could this file be extracted for forms and infolists, or are there any differences between them?

Reply:
We can always refactor later. Focusing on just the header actions makes our lives simpler right now. 🙂